static/js/script.js

```diff
@@ -106,3 +106,26 @@ function filterUsersByHackathon(){
         document.body.removeChild(link);
     });
 }
+
+function enableReviewsSlider(){
+    $('.reviews-slider .next-step').click(function(){
+        let active_elem = $('.reviews-content.active');
+        let next_elem = active_elem.next();
+        if(next_elem && next_elem.hasClass('reviews-content')){
```

Comment:
Missing _space_ before `{`, and _space_ after the `if` at the beginning of line.

Reply:
Added missing spaces.